es-ds-components/components/es-footer.vue

```diff
@@ -0,0 +1,150 @@
+                <div class="row">
+                    <div
+                        class="col col-12 col-md-5 col-lg-4 font-size-200 font-size-md-400 font-weight-bold mb-150 mb-md-0">
+                        {{ content.banner.headline }}
+                        <br />
+                        {{ content.banner.subHeadline }}
+                    </div>
+                    <div class="col col-12 col-md-7 col-lg-8">
+                        {{ content.banner.body }}
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Make an impact -->
+        <div class="bg-blue-800 py-200 pt-lg-200 pb-lg-150 px-200">
+            <div class="container px-0">
+                <div class="row">
+                    <!-- Link columns -->
+                    <div class="col col-12 col-lg-8 order-lg-1 text-white">
+                        <div class="row">
+                            <div
+                                v-for="column in content.columns"
+                                :key="column.title"
+                                class="col col-6 col-md-3 mb-100 mb-lg-0">
+                                <ul class="list-unstyled mb-lg-0">
+                                    <li class="font-weight-bold font-size-75 font-size-md-100 mb-100 mb-lg-50">
+                                        {{ column.title }}
+                                    </li>
+                                    <li
+                                        v-for="link in column.links"
+                                        :key="link.text"
+                                        class="mt-0 mt-lg-25 mb-100 mb-lg-50 font-size-75">
+                                        <es-nav-bar-link
+                                            :href="link.url"
+                                            :target="link.newTab ? '_blank' : null"
+                                            class="text-reset">
+                                            {{ link.text }}
+                                        </es-nav-bar-link>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <!-- Link columns -->
+                    <!-- Logo and Social-->
+                    <div class="col col-12 col-lg-4 order-lg-0">
+                        <es-nav-bar-link
+                            class="d-block mb-150 d-flex justify-content-center justify-content-lg-start"
+                            :href="content.home.link">
+                            <span class="sr-only">
+                                {{ content.home.name }}
+                            </span>
+                            <div class="es-footer-logo">
+                                <slot name="logo" />
+                            </div>
+                        </es-nav-bar-link>
+                        <!-- Social icons -->
```

Comment:
Thanks for catching; I totally missed that. It's because we're using `<component :is="">` and Nuxt 3 is finicky with that. Just had to import them and they worked.